config/prod/nf-osi-processing.yaml

```diff
@@ -0,0 +1,28 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v5.yaml
+stack_name: nf-osi-processing
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "NF OSI"
+  # The resource owner
+  OwnerEmail: "sara.gosline@sagebase.org"
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.2xlarge"
+  # (Optional) ID of the base AMI (supported distros: AWS linux)
+  AMIId: "ami-0d051218cbaf3405a"      # AMI with my R packages
```

Comment:
Any reason why you are using this specific ami, maybe provide info in commit message?

Reply:
No, it was just the default.